src/documentation/pages/components/forms/range-slider/_properties.vue

```diff
@@ -0,0 +1,10 @@
+<template lang="pug">
+  <PropertiesComingSoon />
```

Comment:
Range slider does have JS properties. Please, document the `dispose` method following the examples of other chi.js components.